cmd/runhcs/kill.go

```diff
@@ -34,9 +41,21 @@ signal to the init process of the "ubuntu01" container:
 			return errContainerStopped
 		}
 
-		sigstr := context.Args().Get(1)
-		if sigstr == "" {
-			sigstr = "SIGTERM"
+		signalsSupported := false
+		if c.IsHost {
+			uvm, err := hcs.OpenComputeSystem(vmID(c.ID))
```

Comment:
I think you need to defer a `uvm.Close()` to avoid leaking a registered callback handle

Reply:
Done